pom.xml

```diff
@@ -720,7 +720,7 @@
         <plugin>
           <groupId>com.github.spotbugs</groupId>
           <artifactId>spotbugs-maven-plugin</artifactId>
-          <version>4.5.3.0</version>
+          <version>4.7.1.0</version>
```

Comment:
I had been working on getting us to 4.7.0.0. It generated a bunch of new warnings. If this passes, I'm wondering if they rolled some of those changes back a bit for 4.7.1.0, due to them being too aggressive?